lib/pkg.ml

```diff
@@ -74,11 +74,7 @@ let name p = Ok p.name
 
 let with_name p name = { p with name }
 
-let tag p =
-  match (p.tag, p.version) with
-  | Some v, _ -> Ok v
-  | None, None -> Vcs.get () >>= fun r -> Vcs.describe ~dirty:false r
-  | None, Some v -> Ok v
+let tag p = Vcs.get_tag ?tag:p.tag ?version:p.version ()
```

Comment:
Is there any particular reason why you moved this logic from this module to `Vcs` ?

I think that `Vcs` should stick to running VCS commands and converting their output into OCaml types, this logic is specific to dune-release from the look of it.

Reply:
I've moved the inner logic of the function (but left the function itself), since I need to get the tag before having created a package. But I hadn't noticed the `Vcs` pattern of always running commands and converting their output. Where do you think that function might fit into?

Reply:
I've moved it to `Pkg` now (so now there are two functions to get the tag in `Pkg`).